<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scribbler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextEdit" name="textEdit"/>
      <widget class="SvgView" name="svgView"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConvert_to_Handwritten"/>
    <addaction name="actionLoad_Text_from_File"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Font"/>
    <addaction name="actionFont_Editor"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Current_Sheet_as"/>
    <addaction name="actionSave_All_Sheets"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_Current_Sheet"/>
    <addaction name="actionPrint_All"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionShow_ToolBar"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHowTo"/>
    <addaction name="actionLicenses_and_Credits"/>
    <addaction name="actionAbout_Scribbler"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>&amp;Show ToolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Font">
   <property name="text">
    <string>&amp;Load Font</string>
   </property>
  </action>
  <action name="actionFont_Editor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Font Editor</string>
   </property>
  </action>
  <action name="actionSave_Current_Sheet_as">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Save Current Sheet as Image</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionHowTo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;How To</string>
   </property>
  </action>
  <action name="actionAbout_Scribbler">
   <property name="text">
    <string>&amp;About Scribbler</string>
   </property>
  </action>
  <action name="actionConvert_to_Handwritten">
   <property name="text">
    <string>&amp;Ð¡onvert to Handwritten</string>
   </property>
  </action>
  <action name="actionPrint_Current_Sheet">
   <property name="text">
    <string>&amp;Print Current Sheet</string>
   </property>
  </action>
  <action name="actionShow_ToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show ToolBar</string>
   </property>
  </action>
  <action name="actionLicenses_and_Credits">
   <property name="text">
    <string>&amp;Licenses and Credits</string>
   </property>
  </action>
  <action name="actionSave_All_Sheets">
   <property name="text">
    <string>&amp;Save All Sheets</string>
   </property>
  </action>
  <action name="actionPrint_All">
   <property name="text">
    <string>&amp;Print All Sheets</string>
   </property>
  </action>
  <action name="actionLoad_Text_from_File">
   <property name="text">
    <string>&amp;Load Text from File</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SvgView</class>
   <extends>QGraphicsView</extends>
   <header>svgview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
